tools/Customization/DevHome.Customization/Views/DevDriveInsightsPage.xaml

```diff
@@ -0,0 +1,22 @@
+<Page
+    x:Class="DevHome.Customization.Views.DevDriveInsightsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:views="using:DevHome.Customization.Views"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Always"
```

Comment:
>Always [](http://example.com/codeflow?start=55&length=6)

You add the breadcrumb logic now - 

see [https://github.com/microsoft/devhome/pull/2501](https://github.com/microsoft/devhome/pull/2501)

Reply:
I will do it as a separate PR once I check this in.